.github/workflows/ci-no-code-related.yml

```diff
@@ -0,0 +1,53 @@
+name: CI for Code Un-related Changes
```

Comment:
why is this needed?

Reply:
Commit lint is still needed for the code un-related part.

Reply:
Can the `paths` be filtered at each step of the `regular CI`? Our CI is already non-modular enough

Reply:
`paths` can not be used on step or job level. For the modular concerns, I have some ideas that can improve it. 

I'll push some changes later.

Reply:
I did some experiment. Unfortunately, [composite actions](https://docs.github.com/cn/actions/creating-actions/creating-a-composite-action) can't support all the function we need for CI moderation here.